src/neptune/internal/backends/hosted_client.py

```diff
@@ -175,6 +189,10 @@ def create_backend_client(client_config: ClientConfig, http_client: HttpClient)
 
 @cache
 def create_leaderboard_client(client_config: ClientConfig, http_client: HttpClient) -> SwaggerClientWrapper:
```

Comment:
IDE warns me that `HttpClient` doesn't have `session` attribute. Probably we should change the typing to `RequestsClient`.

Reply:
done